src/foraging_gui/RigJsonBuilder.py

```diff
@@ -0,0 +1,681 @@
+        ]
+
+    components['light_sources'] = [
+             d.LightEmittingDiode(
+                name="IR LED",
+                manufacturer=d.Organization.THORLABS,
+                model="M810L5",
+                wavelength=810,
+            )
+        ]
+
+
+    # Mouse Platform
+    ###########################################################################
+    components['mouse_platform']=d.Tube(
+        name="mouse_tube_foraging", 
+        diameter=3.0,
+        diameter_unit=SizeUnit.CM
+        )
+
+
+    # Stimulus devices
+    ###########################################################################
+    if settings['newscale_serial_num_box{}'.format(settings['box_number'])] != '':
+        stage = d.MotorizedStage(
+                    name="NewScaleMotor for LickSpouts",
+                    serial_number=settings['newscale_serial_num_box{}'.format(settings['box_number'])], 
+                    manufacturer=d.Organization.NEW_SCALE_TECHNOLOGIES,
+                    travel=15.0,
+                    travel_unit=SizeUnit.MM,
+                    firmware="https://github.com/AllenNeuralDynamics/python-newscale, branch: axes-on-target, commit #7c17497",
+                    )
+    else:
+        stage = d.MotorizedStage(
+                    name="AIND lick spout stage",
+                    manufacturer=d.Organization.AIND,
+                    travel=30, 
+                    travel_unit=SizeUnit.MM
+                    )      
+
+    if ('AINDLickDetector' in settings['box_settings']) and (settings['box_settings']['AINDLickDetector'] == "1"):
+        lick_spout_name = 'AIND_Lick_Detector'
+    else:
+        lick_spout_name = 'Janelia_Lick_Detector'
+    lick_spouts=[
+        d.RewardSpout(
+            name="{} Left".format(lick_spout_name),
+            side=d.SpoutSide.LEFT,
+            spout_diameter=1.2,
+            solenoid_valve=d.Device(device_type="Solenoid", name="Solenoid Left"),
+            lick_sensor_type=d.LickSensorType("Capacitive")
+        ),
+        d.RewardSpout(
+            name="{} Right".format(lick_spout_name),
+            side=d.SpoutSide.RIGHT,
+            spout_diameter=1.2,
+            spout_diameter_unit=SizeUnit.MM,
+            solenoid_valve=d.Device(device_type="Solenoid", name="Solenoid Right"),
+            lick_sensor_type=d.LickSensorType("Capacitive")
+        ),
+        ]   
+
+    components['stimulus_devices']=[
+        d.RewardDelivery(
+            reward_spouts=lick_spouts,
+            stage_type = stage,
+        ),
+        d.Speaker(
+            name="Stimulus speaker",
+            manufacturer=d.Organization.OTHER,
+        )
+        ]
+
+
+    # DAQS
+    ###########################################################################
+
+    components['daqs']=[
+        d.HarpDevice(
+            name="Harp Behavior",
+            harp_device_type=d.HarpDeviceType.BEHAVIOR,
+            manufacturer=d.Organization.CHAMPALIMAUD,
+            core_version="2.1",
```

Comment:
the `core_version` is not necessarily "2.1". Most of them are 2.0. We should check for each behavior board.

Reply:
They aren't all the same!? That seems like something we should fix

Reply:
this is also new to me...thought all the same. I can check ones in 446/447/428 tomorrow. (and maybe ask Helio)

Reply:
Do you know how to check the version in code?

Reply:
I removed core_version, and firmware_version. They are listed as optional, and since they might be different I wont set them unless we can set them automatically

Reply:
I don't know how to check the version through bonsai code. It's ok to remove the core_version at this moment.

Reply:
Bruno gave me some tips on how to get the core version, firmware version, and hardware version in code. I'll make a separate issue